jetty-ee10/jetty-ee10-osgi/pom.xml

```diff
@@ -67,9 +68,11 @@
       </resource>
     </resources>
     <plugins>
+      <!-- is it really used?? this plugiin has been abandonned and no version since 2012... -->
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-eclipse-plugin</artifactId>
+        <version>2.9</version>
```

Comment:
@janbartel do you really use this plugin?

Reply:
@janbartel ping? maybe we can remove those lines?

Reply:
@olamy nope, I don't use that plugin at all, so go ahead and remove.